### DISCUSSION TITLE
New Source fields (BT-806 and BT-821) in SDK 1.12

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#1000] [Q&A] [answered]
In sdk 1.12 the new "source" field has been added for the BT-806 and BT-821 nodes.
In which cases should the "epo-notice" value and the "epo-procurement-document" value be indicated in the BT-806 and BT-821 nodes in the SDK 1.12?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
"epo-notice" is used to indicate that the corresponding criteria will be found in the notice itself. When the notice is identified as a source for criteria, then these criteria have to be specified in the notice.
"epo-procureent-document" is used to indicate that the corresponding criteria are defined in the Procurement Document.
KR
